### PR TITLE
bump slickgrid to take a hot fix

### PR DESCRIPTION
**Pull request by anthonydresser**
Bump slickgrid to take https://github.com/anthonydresser/SlickGrid/commit/e7783f6a26b0ab0c34997a94caf2cefc186e1b77 which patches a bug that was introduced with the slickgrid downward merge. It would arise when restore was opened before a edit/result window was opened.

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -134,9 +134,9 @@
       "resolved": "https://registry.npmjs.org/buffer-crc32/-/buffer-crc32-0.2.13.tgz"
     },
     "caniuse-db": {
-      "version": "1.0.30000793",
+      "version": "1.0.30000795",
       "from": "caniuse-db@>=1.0.30000161 <2.0.0",
-      "resolved": "https://registry.npmjs.org/caniuse-db/-/caniuse-db-1.0.30000793.tgz"
+      "resolved": "https://registry.npmjs.org/caniuse-db/-/caniuse-db-1.0.30000795.tgz"
```
Comment (MattIrv) [inline npm-shrinkwrap.json]: Is this related to the slick grid fix?

Comment (anthonydresser) [reply npm-shrinkwrap.json]: no, this is just the version that npm pulls without a shrinkwrap based on our current package.json (can't wait for the 1.19 vscode merge)
